ui/app/mainui/activitycenter/popups/ActivityCenterPopup.qml

```diff
@@ -369,4 +402,88 @@ Popup {
             onCloseActivityCenter: root.close()
         }
     }
+
+    Component {
+        id: pairDeviceDialog
+
+        StatusDialog {
+            property string name
+            property string deviceId
+
+            width: 620
+            closePolicy: Popup.CloseOnPressOutside
+
+            title: qsTr("Pair new device and sync profile")
+
+            contentItem: ColumnLayout {
+                spacing: 16
+                StatusBaseText {
+                    Layout.fillWidth: true
+                    font.pixelSize: 15
+                    text: qsTr("New device with %1 profile has been detected. You can see the device ID below and on your other device. Only confirm the request if the device ID matches.")
+                        .arg(name)
+                    wrapMode: Text.WordWrap
+                }
+                StatusBaseText {
+                    Layout.alignment: Qt.AlignHCenter
+                    font.pixelSize: 27
+                    font.weight: Font.Medium
+                    text: deviceId
+                }
+            }
+
+            footer: StatusDialogFooter {
+                leftButtons: ObjectModel {
+                   StatusFlatButton {
+                        text: "Cancel"
+                        onClicked: {
+                            close()
+                        }
+                    }
+                }
+                rightButtons: ObjectModel {
+                    StatusButton {
+                        text: "Pair and Sync"
+                        onClicked: {
+                            activityCenterStore.enableInstallationAndSync(deviceId)
+                            close()
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    Component {
+        id: checkOtherDeviceDialog
+
+        StatusDialog {
+            property string deviceId
+
+            width: 620
+            closePolicy: Popup.CloseOnPressOutside
+
+            title: qsTr("Pair this device and sync profile")
+
+            contentItem: ColumnLayout {
+                spacing: 16
+                StatusBaseText {
+                    Layout.fillWidth: true
+                    font.pixelSize: 15
+                    text: qsTr("Check your other device for a pairing request. Ensure that the this device ID displayed on your other device. Only proceed with pairing and syncing if the IDs are identical.")
+                    wrapMode: Text.WordWrap
+                }
+                StatusBaseText {
+                    Layout.alignment: Qt.AlignHCenter
+                    font.pixelSize: 27
+                    font.weight: Font.Medium
+                    text: deviceId
+                }
+                Item {
+                    Layout.fillWidth: true
+                }
+            }
+            footer: null
```

Comment:
Does this really have no buttons? :thinking:

Reply:
Yup https://www.figma.com/design/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=40765-401872&node-type=frame&t=dGj7XVHOMQlL3Juw-0